internal/market/coin.go

```diff
@@ -43,11 +50,14 @@ func (c Coin) String() string {
 }
 
 // IsAvailableForTrading checks if the coin should be picked up by the bot for trading.
-// It checks if the coin is in the custom list and if it's not a fiat currency, both of which are options defined in the given config.
-func (c Coin) IsAvailableForTrading(allowList, denyList []string, pairWith string) bool {
+// It checks whether the coin has the desired minimum quote asset trading volume, is in the custom list, and it's not a blacklisted symbol. These options are defined in the given config file.
+func (c Coin) IsAvailableForTrading(allowList, denyList []string, pairWith string, minQuoteVolumeTraded float64) bool {
+	if c.QuoteVolumeTraded < minQuoteVolumeTraded {
```

Comment:
If the `min_quote_volume_traded` option is not defined in the config, the `minQuoteVolumeTraded` value will default to go's zero value, which is `0.0` in this case.  I think it's be safe to assume the user wants to turn this trading feature off whenever that's the case. Thus I'd suggest adding a check for this like:

```go
if minQuoteVolumeTraded != 0.0 &&  c.QuoteVolumeTraded < minQuoteVolumeTraded {
    // ...
}
```

Reply:
good point, I was assuming that all coins would have a 24H traded volume > 0 after fetching the volumes, so they'd typically all be greater than the default 0.0 value, but there might be cases where the volume is actually 0 for the period and they might be inadvertently filtered out, or in case we somehow fail to retrieve the volume for a coin. It's better not to assume anything and check explicitly.